src/output.rs

```diff
@@ -1,37 +1,47 @@
 use derive_new::new;
 use std::io::Write;
 
-use crate::dial::DialId;
-
+/// A constant for the CSV file headers
 const CSV_HEADERS: &str = "alarm, rms_error, response_time, correct_key, key";
 
+/// A struct that helps to collect AlarmReactions and can output them to a CSV file
 pub struct SessionOutput {
-    pub dial_reactions: Vec<DialReaction>,
+    /// The current alarm reactions in chronological order
+    pub alarm_reactions: Vec<AlarmReaction>,
+    /// The output path to the CSV
     pub output_path: String,
 }
 
-/// Information about a user's responce to an instance of an alarm being fired
-#[derive(Debug, Copy, Clone, new)]
-pub struct DialReaction {
-    pub id: DialId,
+/// Information about a user's response to an instance of an alarm being fired
+#[derive(Debug, Clone, new)]
+pub struct AlarmReaction {
+    /// The name of the alarm that the dial reaction was to
+    pub alarm_name: String,
+    /// The reaction time to the alarm in milliseconds
     pub millis: u32,
```

Comment:
Would it be better to store this as a `Duration` and use `to_millis` later? If it comes from summing delta times that are not already a duration then I think this is fine.

Reply:
The only reason this struct exists is to be written to the CSV file. I think it should contain the values that we are going to write directly, since its sole purpose is output. It does come from `elapsed().to_millis()`, but we never need anything else from the duration at all except for the millis to output.

Reply:
Fair enough